crates/storage/cache/cache.rs

```diff
@@ -0,0 +1,40 @@
+use error::MemoryDBError;
+use revm::{
```

Comment:
We don't want storage crates to have a dependency on `revm`, because we want to be able to switch evm implementations. So it should not leak outside the `evm` crate.

Reply:
got it, will move this to the `vm` crate, thanks!